ext/ffi/lib.rs

```diff
@@ -2017,6 +2157,20 @@ fn op_ffi_get_static<'scope>(
       };
       integer.into()
     }
+    NativeType::Struct(fields) => {
```

Comment:
I'm not sure it makes sense to speak of a static struct...
1. It's by definition accessed through a pointer.
2. Copying or getting mutable access to a buffer / struct through a pointer is already possible using the `Deno.UnsafePointerView#getArrayBuffer()` and `Deno.UnsafePointerView#copyInto()` static methods.

The only use I can see for this is the automatic size computation but that's not hugely beneficial as the user still needs to know the internal layout of the struct to access data from the `Uint8Array` (If this returned an Array then that'd be different, but that would be a huge difference to all other related APIs.)